.travis.yml

```
@@ -71,7 +71,8 @@ script:
   # Check for clang format
   - cd src/avoidance
   - git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-  - ./check_code_format.sh
+  - pushd tools && ./check_code_format.sh && popd
```

So when `check_code_format.sh` fails, `popd` won't be called and you end up in the wrong directory, right?

I would instead suggest:

```suggestion
  - (cd tools && ./check_code_format.sh)
```

---

The CI then fails due to the `set -e`, but yes this is still better.

---

I see, that works too.